src/components/sidenav/sidenav.spec.js

```diff
@@ -85,11 +84,11 @@ describe('mdSidenav', function() {
     it('should focus child with md-sidenav-focus', inject(function($rootScope, $material, $document, $compile) {
       jasmine.mockElementFocus(this);
       var parent = angular.element('<div>');
-      var markup = '<md-sidenav md-is-open="show">'+
-                      '<md-input-container><label>Label</label>' +
-                        '<input type="text" md-sidenav-focus>' +
-                      '</md-input-container>' +
-                    '<md-sidenav>';
+      var markup = '<md-sidenav md-is-open="show">' +
+        '<md-input-container><label>Label</label>' +
```

Comment:
Can you keep the indents here? Or add the spaces within the quotes so that your formatter won't mess it up?